src/app/modules/results-browser/results-browser/results-browser.component.scss

```diff
@@ -41,9 +41,12 @@
         height: 9rem;
         width: 95%;
         bottom: 0;
-        // background: linear-gradient(0deg, rgba(33,33,33,1) 0%, rgba(33,33,33,1) 15%, rgba(33,33,33,0) 100%);
     }
     .scroll-gradient.hidden {
         display: none;
     }
+}
+
+.highlighted {
+    background-color: #92adc7;
```

Comment:
Extract to theme. I know I said to pick a color before, but i think the blue used to highlight other things (like the selected ontology), etc. is probably the better color to use.

Reply:
Oh and you know what, let's just color the title "Male, Age XX, BMI YY". More subtle but less disruptive.